commands/publish/__tests__/publish-canary.test.js

```diff
@@ -14,7 +14,7 @@ const childProcess = require("@lerna/child-process");
 
 // mocked modules
 const writePkg = require("write-pkg");
-const npmPublish = require("@lerna/npm-publish");
+const npmDistTag = require("@lerna/npm-dist-tag");
```

Comment:
Most assertions on the final dist-tag now need to use the `@lerna/npm-dist-tag` mock, which explains all the new stuff in the mock helper. (`@lerna/npm-publish` is almost always called with `lerna-temp` dist-tag, which isn't very useful as an assertion)